rust/links/src/router.rs

```diff
@@ -72,6 +74,10 @@ where
         .layer(cors)
         .with_state(state);
 
-    let recorder_handle = setup_metrics_recorder();
-    router.route("/metrics", get(move || ready(recorder_handle.render())))
+    if enable_metrics {
```

Comment:
Maybe should just return router at the end at all times? Im on the "hate if/else" gang